utils/api/base.js

```diff
@@ -11,6 +11,7 @@ export const fetcher = (url, token) => {
     .then(res => res.data)
     .catch(error => {
       Sentry.captureException(error)
+      throw error
```

Comment:
without this, we are only getting the placeholder cards on the homepage, instead of the error notice.

if we use `return` instead of `throw`, then the useSWR hook would incorrectly get the error as the data property:
``` javascript
{
  data: <AxiosError...>,
  error: undefined
}
```